### PR TITLE
Add splinter upgrade command manpage

### PR DESCRIPTION
**Pull request by leebradley**
Add manpage with a usage of the new `splinter upgrade` command

Signed-off-by: Lee Bradley <bradley@bitwise.io>

Depends on https://github.com/Cargill/splinter/pull/1363

### CODE REVIEW

File: cli/man/splinter-upgrade.1.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+**splinter-upgrade** — Upgrades splinter yaml state to database state
+
+SYNOPSIS
+========
+
+**splinter** **upgrade** \[**FLAGS**\]
+
+DESCRIPTION
+===========
+Upgrades splinter by importing data from the deprecated yaml state file format
```
Comment (agunde406) [inline cli/man/splinter-upgrade.1.md:20]: Can you add some information here about how the files are renamed? Also add an EXAMPLE and SEE ALSO section

**Review by agunde406 — CHANGES_REQUESTED** [review]

File: cli/man/splinter-upgrade.1.md (hunk 1 above)
Comment (leebradley) [reply cli/man/splinter-upgrade.1.md:20]: Updated!

File: cli/man/splinter.1.md [hunk 2]
```diff
@@ -55,6 +55,9 @@ SUBCOMMANDS
 `registry`
 : Provides commands to create and manage Splinter registry information.
 
+`upgrade`
```
Comment (agunde406) [inline cli/man/splinter.1.md:58]: upgrade should also be added to the SEE ALSO section below

**Review by agunde406 — CHANGES_REQUESTED** [review]

File: cli/man/splinter.1.md (hunk 2 above)
Comment (leebradley) [reply cli/man/splinter.1.md:58]: Done!

**Review by peterschwarz — APPROVED** [review]

**Pull request merged**